jax/_src/test_util.py

```diff
@@ -1433,10 +1433,21 @@ def supported(self, dtypes):
 
   @_cached_property
   def custom_floats(self):
-    return [np.dtype(t) for t in [
-      _dtypes.bfloat16, _dtypes.float8_e4m3b11fnuz,
-      _dtypes.float8_e4m3fn, _dtypes.float8_e4m3fnuz,
-      _dtypes.float8_e5m2, _dtypes.float8_e5m2fnuz]]
+    float_dtypes = [
+      _dtypes.bfloat16,
+      _dtypes.float8_e4m3b11fnuz,
+      _dtypes.float8_e4m3fn,
+      _dtypes.float8_e4m3fnuz,
+      _dtypes.float8_e5m2,
+      _dtypes.float8_e5m2fnuz,
+    ]
+    # TODO: Remove once minimum_jaxlib_version is 0.4.35+
+    if jax.version._minimum_jaxlib_version >= "0.4.35":
```

Comment:
Similar here: rather than testing against `minimum_jaxlib_version`, we need to test against the actual installed jaxlib version.

Reply:
fixed